src/treemodel.cpp

```diff
@@ -457,8 +457,12 @@ void TreeModel::SetupChild(TreeItem *child, const QJsonObject & event)
     bool hasArtData = event.contains("a");
     if (hasArtData)
     {
-        // u25CF = BLACK CIRCLE
+        // u25CF = BLACK CIRCLE; Use QString with MSVC to avoid locale dependent C4566 warning.
+#ifdef Q_CC_MSVC
```

Comment:
Do we really need that ifdef? can't we always use the  QString::fromUtf16(u"\u25CF")?

Reply:
I haven't checked but fromUtf16() should work on OSX as well. The reason I keep QStringLiteral with #ifdef, is because it is the recommended efficient approach per documentation, and using fromUtf16 is more of a workaround to the MSVC only compiler warning.